app/src/org/commcare/tasks/CleanRawMedia.kt

```diff
@@ -0,0 +1,31 @@
+package org.commcare.tasks
+
+import android.content.Context
+import androidx.work.Worker
+import androidx.work.WorkerParameters
+import org.commcare.CommCareApplication
+import org.commcare.android.database.user.models.FormRecord
+import org.commcare.utils.CrashUtil
+import org.commcare.utils.StorageUtils.getUnsentCompleteOrSavedFormIdsForCurrentApp
+import java.io.File
+
+class CleanRawMedia(appContext: Context, workerParams: WorkerParameters)
+    : Worker(appContext, workerParams) {
+
+    override fun doWork(): Result {
+        val storage = CommCareApplication.instance().getUserStorage(FormRecord::class.java)
+        val recordsToRemove = getUnsentCompleteOrSavedFormIdsForCurrentApp(storage)
+        for (formId in recordsToRemove) {
+            try {
+                val filePath = storage.getMetaDataFieldForRecord(formId, FormRecord.META_FILE_PATH)
+                File(filePath).parent?.let {
+                    val rawDirPath = "$it/raw"
+                    File(rawDirPath).deleteRecursively()
+                }
+            } catch (e: Exception) {
+                CrashUtil.reportException(e)
+            }
+        }
+        return Result.success()
+    }
+}
```

Comment:
NIT: Missing endline

Reply:
Ahh I am not sure what are you pointing at here ? Is it nicer to give a line break at the end of file ?

Reply:
Found an explanation [here](https://stackoverflow.com/a/729795/3811963), so sorted on this. Configured my editor to do this in future for all files.

Reply:
Wow, I didn't know about this. Will configure my editor for the same.